### ISSUE TITLE
[calib2] '.' in the input text cause some troubles.

### ISSUE THREAD

**Issue by picca** [silx-kit/pyFAI#975]
Hello, when I try to change an entry like the wavelength and press the '.' the cursor doesn not position itself after the dot, but before.

So instead of

1.54

I have

154.

**Comment by vallsv** [comment]
Ok, i see, you start from an empty widget.

**Comment by picca** [comment]
Yes :)

**Comment by picca** [comment]
Hello, here the version I am using on DIFFABS (Debian stable )

```
:/nfs/ruche-diffabs/diffabs-users/20170063/published-data/xrd/calibration$ dpkg -l | grep pyfai
ii  pyfai                                             0.16.0+dfsg1-2                              all          Fast Azimuthal Integration scripts
ii  python-pyfai                                      0.16.0+dfsg1-2                              amd64        Fast Azimuthal Integration scripts - Python2
ii  python-pyfai-doc                                  0.16.0+dfsg1-2                              all          Fast Azimuthal Integration scripts - Documentation
ii  python3-pyfai                                     0.16.0+dfsg1-2                              amd64        Fast Azimuthal Integration scripts - Python3
reguer@diffabs6:/nfs/ruche-diffabs/diffabs-users/20170063/published-data/xrd/calibration$ dpkg -l | grep pyqt5
ii  python-pyqt5                                      5.7+dfsg-5                                  amd64        Python 2 bindings for Qt5
ii  python-pyqt5-dbg                                  5.7+dfsg-5                                  amd64        Python 2 bindings for Qt5 (debug extensions)
ii  python-pyqt5.qtmultimedia                         5.7+dfsg-5                                  amd64        Python 2 bindings for Qt5's Multimedia module
ii  python-pyqt5.qtopengl                             5.7+dfsg-5                                  amd64        Python 2 bindings for Qt5's OpenGL module
ii  python-pyqt5.qtopengl-dbg                         5.7+dfsg-5                                  amd64        Python 2 bindings for Qt5's OpenGL module (debug extension)
ii  python-pyqt5.qtsvg                                5.7+dfsg-5                                  amd64        Python 2 bindings for Qt5's SVG module
ii  python-pyqt5.qtsvg-dbg                            5.7+dfsg-5                                  amd64        Python 2 bindings for Qt5's SVG module (debug extension)
ii  python-pyqt5.qtwebkit                             5.7+dfsg-5                                  amd64        Python 2 bindings for Qt5's WebKit module
ii  python3-pyqt5                                     5.7+dfsg-5                                  amd64        Python 3 bindings for Qt5
ii  python3-pyqt5-dbg                                 5.7+dfsg-5                                  amd64        Python 3 bindings for Qt5 (debug extensions)
ii  python3-pyqt5.qtmultimedia                        5.7+dfsg-5                                  amd64        Python 3 bindings for Qt5's Multimedia module
ii  python3-pyqt5.qtopengl                            5.7+dfsg-5                                  amd64        Python 3 bindings for Qt5's OpenGL module
ii  python3-pyqt5.qtopengl-dbg                        5.7+dfsg-5                                  amd64        Python 3 bindings for Qt5's OpenGL module (debug extension)
ii  python3-pyqt5.qtsvg                               5.7+dfsg-5                                  amd64        Python 3 bindings for Qt5's SVG module
ii  python3-pyqt5.qtsvg-dbg                           5.7+dfsg-5                                  amd64        Python 3 bindings for Qt5's SVG module (debug extension)
ii  python3-pyqt5.qtwebkit                            5.7+dfsg-5                                  amd64        Python 3 bindings for Qt5's WebKit module
```

**Comment by vallsv** [comment]
Looks to be an issue which have to be fixed on the 0.16 branch.
